src/api/notionApi.js

```diff
@@ -0,0 +1,47 @@
+import request from './index.js';
+
+const STATIC_URL = 'documents';
+
+const createDocument = async id => {
+  try {
+    return await request(`documents/${id}`);
+  } catch (error) {
+    console.error(error.message);
+  }
+};
```

Comment:
```suggestion
const createDocument = async (id, body) => {
  try {
    return await request(`documents/${id}`, {
      method: 'POST',
      body: JSON.stringify(body), // {'title': '', 'parent': null});
  } catch (error) {
    console.error(error.message);
  }
};
```
request body가 필요할 것 같습니다. POST로 parent 아래에 넣어야 된다고 생각하기 때문입니다 😊